Add 4.5.1 changelog (#6339)
wazuh/wazuh-documentation: CHANGELOG.md

# Change Log
All notable changes to this project will be documented in this file.

> ## [v4.5.1] – TBD

Reviewer: ```suggestion
## [v4.5.1]
```

### Added

> - Modify the documentation including the new macOS Apple silicon installers ([#6296](https://github.com/wazuh/wazuh-documentation/pull/6296))

Reviewer: ```suggestion
- Included the new macOS Apple silicon installers. ([#6296](https://github.com/wazuh/wazuh-documentation/pull/6296))
```

> - Add discard_regex parameter to Services ([#6207](https://github.com/wazuh/wazuh-documentation/pull/6207))

Reviewer: ```suggestion
- Added `discard_regex` parameter to the services section of the AWS module reference. ([#6207](https://github.com/wazuh/wazuh-documentation/pull/6207))
```

> - [4.5.1] Add support for Kibana 7.17.10 and 7.17.11 ([#6311](https://github.com/wazuh/wazuh-documentation/pull/6311))

Reviewer: ```suggestion
- Added support for Kibana 7.17.10 and 7.17.11. ([#6311](https://github.com/wazuh/wazuh-documentation/pull/6311))
```

> - Add 4.5.1 release notes ([#6309](https://github.com/wazuh/wazuh-documentation/pull/6309))

Reviewer: ```suggestion
- Added the Wazuh v4.5.1 release notes. ([#6309](https://github.com/wazuh/wazuh-documentation/pull/6309))
```
